lib/libutee/arch/arm/utee_syscalls_a32.S

```diff
@@ -52,7 +60,19 @@
         .endif
         svc #0
         pop     {r5-r7,pc}
+UNWIND( .fnend)
 	END_FUNC \name
         .endm
 
+	FUNC utee_panic, :
+UNWIND( .fnstart)
+	adr	r11, .
+	push	{r0-r11, lr}
```

Comment:
Why not push `pc` instead of passing it via `r11`? There's also the risk that the original value of `r11` is needed for stack unwinding.

Reply:
Fixed.